pkg/test/component_mysql.go

```diff
@@ -40,6 +40,10 @@ func (m *mysqlComponentLegacy) Start() error {
 		"MYSQL_ROOT_PASSWORD=gosoline",
 	}
 
+	if len(m.settings.Tmpfs) == 0 {
+		m.settings.Tmpfs["/var/lib/mysql"] = ""
```

Comment:
This doesn't seem to have anything with the wiremock to do. Why do we need that? Can we add a comment here?

Reply:
you are right, it is only related to the mysql mock (the old one) and is basically making both mysql mocks act the same. The new one does also have this tmpfs by default.